week1/1-class.js

```diff
@@ -103,19 +110,35 @@ class Cities {
     this.#list.sort((d1, d2) => d2.density - d1.density);
   }
 
-  print({ addDensityPercentage = true } = {}) {
-    this.#list.forEach((city) => {
-      city.print({ addDensityPercentage, maxDensity: this.#maxDensity });
-    });
+  forEach(callback) {
+    for (let city of this.#list) {
```

Comment:
Prefer `const` over `let` when the variable is not reassigned. It signals immutability and improves code clarity.
```suggestion
    for (const city of this.#list) {
```